test/controllers/devise_token_auth/confirmations_controller_test.rb

```diff
@@ -127,7 +127,7 @@ def token_and_client_config_from(body)
         test 'bad request on resend confirmation' do
           post :create, params: { email: nil }, xhr: true
 
-          assert_equal 400, response.status
+          assert_equal 401, response.status
```

Comment:
why this change? bad request is 400

Reply:
It was 401 in [unlock_controller](https://github.com/lynndylanhurley/devise_token_auth/blob/f39244eabcfd58f0fa31421774fd10de569baea4/app/controllers/devise_token_auth/unlocks_controller.rb#L60). That's why I changed it. I think we can change the test description to `request resend confirmation without email`. What do you think?

Reply:
oh cool, thanks